components/MapPage.qml

```diff
@@ -30,6 +30,14 @@ PageWithBottomEdge {
 
         head.actions: [
             Action {
+                id: navigateAction
```

Comment:
I think it's better to keep "where am I" action before, because I feel it's one of the actions you want to access in one click only.